[bitnami/postgresql-ha] Fix serviceLabels error in postgresql-ha chart

bastienbosser opened the pull request:
### Description of the change

This PR fix the bug on serviceLabels in postgresql-ha chart.

### Benefits

We are now able to add your own labels in postgresql service.

### Applicable issues

  - fixes #11451

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

marcosbc reviewed (changes requested):
Hi @bastienbosser, my understanding is that the `service.serviceLabels` option is the one meant for the PostgreSQL service, which is declared in the file that is being modified. This is how it looks in `values.yaml` (NOTE: there seems to be a typo in the `service:` section, as it should be saying `PostgreSQL` and not `PgPool`:

```
## PgPool service parameters
##
service:
  ...
  ## @param service.serviceLabels Labels for PostgreSQL service
  ##
  serviceLabels: {}
```

Therefore, I think that the current implementation is correct, just that the `values.yaml` is wrongly described. Would you mind fixing this issue instead in the PR?

bastienbosser commented:
Hi @marcosbc,
I agree with you there is an error in the values.yaml file and I just corrected it.
I also corrected the error I had made in the if of the new implementation I propose so that we can add labels to the postgresql service. Indeed, for me the implementation in place in the last release is not functional #11451.

marcosbc reviewed (approved):
LGTM! Thanks for the fix!
